gateway/conf/nginx.conf.liquid

```diff
@@ -120,7 +120,8 @@ http {
 
   server {
 
-    access_log {{ access_log_file | default: "/dev/stdout" }} time;
+    set $enable_access_logs '1';
```

Comment:
I'd rather name it `access_logs_enabled` so we can later have `access_logs_format` etc.

Reply:
Sounds good :+1: